Client/Frontend/UserContent/UserScripts/FingerprintingProtection.js

```diff
@@ -13,15 +13,7 @@ if (webkit.messageHandlers.$<handler>) {
         function installIFrameMethodTrap(obj, method) {
             let orig = obj[method];
             obj[method] = function () {
-                var last = arguments[arguments.length - 1]
-                if (last && last.toLowerCase() === 'canvas') {
-                    // Prevent fingerprinting using contentDocument.createElement('canvas'),
-                    // which evades trapInstanceMethod when the iframe is sandboxed
-                    sendMessage({ obj: `${obj}`, method: method })
-                } else {
-                    // Otherwise apply the original method
-                    return orig.apply(this, arguments)
-                }
+                return orig.apply(this, arguments)
             };
             return orig;
```

Comment:
This only removes the protection on making a `canvas` element using `createElement`/`createElementNS`.  In fact if thats all thats needed this whole function can be removed and the hooks removed where `installIFrameMethodTrap` is called.

There are two other `<canvas>` element method hooks in place (`toBlob` and `toDataURL`), do these also need to be removed?

Reply:
Good catch @kylehickinson! I just updated the PR removing both `toBlob` and `toDataURL` hooks.